packages/react-server-module-tagger/index.js

```diff
@@ -49,81 +8,122 @@ module.exports = function(fullMatch, optString){
  *         { label: 'sub' },
  *         'my-project.src',
  *         'my-project/src/components/my-component.js'
- *     ) // returns "components.my-component"
+ *     ) // returns 'components.my-component'
  * @param  {String}   fn       filename
  * @param  {Object}   opts     { label: 'Optional logger label' }
  * @param  {String}   trim     The leading portion of the name to remove.
  * @param  {String}   basePath The path to the file
  * @return {String}            The logger name, e.g. my-project.components.my-component
  */
-var getName = function(fn, opts, trim, basePath){
-	var slashPattern = isWindows
-		?/\\/g
-		:/\//g
-	var name = fn.substring(basePath.length+trim.length, fn.length)
+var getName = function (fn, opts, trim, basePath) {
+	var slashPattern = isWindows ? /\\/g : /\//g;
+	var name = fn.substring(basePath.length + trim.length, fn.length)
 		.replace(/\.jsx?$/, '')
-		.replace(slashPattern,'.')
+		.replace(slashPattern, '.');
 	if (opts.label) {
-		name += '.'+opts.label
+		name += '.' + opts.label;
 	}
 	return name;
-}
+};
 
 /**
  * Gets isomorphic color objects from filenames.
  * @param  {String} filename
- * @return {Object} An isomorphic color object in the form {"color":{"server":147,"client":"rgb(127,127,212)"}}
+ * @return {Object} An isomorphic color object in the form {'color':{'server':147,'client':'rgb(127,127,212)'}}
  */
-var getColor = (function(){
-
+var getColor = (function () {
 	// ANSI escape sequence on the server.
 	// CSS rgb(...) color in the browser.
-	var makeColor = function(r,g,b){
+	var makeColor = function (r, g, b) {
 		return {
-			server: 16 + r*36 + g*6 + b,
-			client: "rgb("+[
-				(r*42.5)|0,
-				(g*42.5)|0,
-				(b*42.5)|0,
-			].join(',')+")",
-		}
-	}
+			server: 16 + r * 36 + g * 6 + b,
+			client: 'rgb(' + [
+				(r * 42.5) | 0,
+				(g * 42.5) | 0,
+				(b * 42.5) | 0
+			].join(',') + ')'
+		};
+	};
 
 	// This produces a list of 24 colors that are distant enough from each
 	// other to be visually distinct.  It's also, conveniently, the same
 	// palette client side and server side.
 	var colors = [];
-	for (var r = 1; r < 6; r+=2) {
-		for (var g = 1; g < 6; g+=2) {
-			for (var b = 1; b < 6; b+=2) {
+	for (var r = 1; r < 6; r += 2) {
+		for (var g = 1; g < 6; g += 2) {
+			for (var b = 1; b < 6; b += 2) {
 				if (r !== g || g !== b) { // No gray.
-					colors.push(makeColor(r,g,b));
+					colors.push(makeColor(r, g, b));
 				}
 			}
 		}
 	}
 
-
 	// Just want a fairly well distributed deterministic mapping.
 	//
 	// Adapted from:
 	// http://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript-jquery
-	var hash = function(str){
-		var hash = 0, i, chr, len;
-		if (str.length === 0) return hash;
-		for (i = 0, len = str.length; i < len; i++) {
-			chr   = str.charCodeAt(i);
-			hash  = ((hash << 5) - hash) + chr;
+	var hash = function (str) {
+		var hash = 0;
+		var chr;
+		if (str.length === 0) {
+			return hash;
+		}
```

Comment:
Its what we use for [the generator](https://github.com/redfin/react-server/blob/master/packages/generator-react-server/package.json#L31) and [the bike share example](https://github.com/redfin/react-server/blob/master/packages/react-server-examples/bike-share/package.json#L12) so I figured it wouldn't be an issue.  Mostly I like it because its really easy to set up.